helm/net-exporter-chart/templates/daemonset.yaml

```diff
@@ -25,11 +25,16 @@ spec:
           runAsUser: 1000
           runAsGroup: 1000
       priorityClassName: giantswarm-critical
+      {{ if or .Values.debug.nicExporter .Values.debug.nstatExporter }}
+      hostNetwork: true
+      {{ end }}
```

Comment:
I am afraid this will break net-exporter function, as net=exporter is testing pod-to-pod network connectivity by pinging accessing the pod IP. If you enable Host network than this functionality is removed as pod will have node ip

Reply:
that's true. Well, then the only way is to keep it separately